src/modules/elixir.rs

```diff
@@ -0,0 +1,81 @@
+use ansi_term::Color;
+use lazy_static::lazy_static;
+use regex::Regex;
+use std::process::Command;
+
+use super::{Context, Module};
+
+const ELIXIR_VERSION_PATTERN: &str = "\
+Erlang/OTP (?P<otp>\\d+)[^\\n]+
+
+Elixir (?P<elixir>\\d[.\\d]+).*";
+
+lazy_static! {
```

Comment:
Does using `lazy_static` gain us any advantages over just initializing the regex at the start of the `module` function? Keep in mind that each prompt draw of starship corresponds to a new process, so we can only reuse lazy statics if we use this variable more than once per prompt draw.

Reply:
Forgot the process run every time, so yeah this line is totally useless actually, i will remove it and and add the regex where it's actually used.